pkg/workload/workload.go

```diff
@@ -454,6 +454,18 @@ func HasRetryOrRejectedChecks(wl *kueue.Workload) bool {
 	return false
 }
 
+// Returns the list of Rejected admission checks
+func GetRejectedChecks(wl *kueue.Workload) []string {
```

Comment:
I would suggest placing this function is some utility package for admission checks.

Reply:
Let's create the utility package with, or after #1174